src/main/webapp/app/trials/views/trials.create.html

```diff
@@ -6,42 +6,42 @@
 
```

Comment:
Just thinking Stephen, the refactor I'm going for the TrialInfo will remove the trials.create.html so you can ignore updating this file.